src/Utils/samples.jl

```diff
@@ -198,73 +275,5 @@ function _sample_unit_nball_muller!(D::Vector{Vector{N}}, n::Int, p::Int;
     return D
 end
 
-# =====================
-# Rejection Sampling
-# =====================
-
-"""
-    RejectionSampler{S<:LazySet, D<:Distribution} <: Sampler
```

Comment:
`Distribution` can now be removed from this line:
`using .Distributions: Distribution, Uniform, Normal`